scalability_and_performance/recommended-cluster-scaling-practices.adoc

```diff
@@ -5,6 +5,14 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+
+[IMPORTANT]
+====
+The guidance in this section is only relevant for installations with cloud
+provider integration. Currently, there is no MachineSet available with bare
+metal installations.
```

Comment:
Machinesets are available withing Baremetal deployments, but it is just that we need to add the baremetal host objects before scaling up machinesets

Reply:
For example, a baremetal host object needs to be added using a manifest like https://github.com/metal3-io/baremetal-operator/blob/master/examples/worker-0.yaml before the replicaset is scaled in baremetal.

Reply:
Thanks for the pointers @smalleni.